constants.js

```diff
@@ -128,6 +131,7 @@ module.exports.pullRequestReviewEvent = pullRequestReviewEvent;
 module.exports.checkCompletedEvent = checkCompletedEvent;
 module.exports.periodicCheckEvent = periodicCheckEvent;
 module.exports.issueCommentCreatedEvent = issueCommentCreatedEvent;
+module.exports.ensureAllIssuesHaveProject = ensureAllIssuesHaveProject;
```

Comment:
Since this is run when an issue is opened, should this be `ensureNewIssuesHaveProjects`?

Reply:
Sir Actually I have a doubt to clarify here, Current the function  which I am using(Was written already),WIll comment on all the issues which do not have project assigned, Do you want that this should be restricted to only newly opened issues. Please let me know, So I can proceed in this PR.

Reply:
Yep, I think we should only check the issue that triggered the event.